bioinf-tools-1/add_modules/dna_rna_tools_module.py

```diff
@@ -0,0 +1,42 @@
+nucl_dna = {"a", "t", "g", "c", "A", "T", "G", "C"}
+nucl_rna = {"a", "u", "g", "c", "A", "U", "G", "C"}
+    
+def is_na(seq: str) -> bool:
+	    return set(seq) <= (nucl_dna) or set(seq) <= (nucl_rna)
+
+
+transcribe_dict = {"T": "U", "t": "u"}
+
+
+def transcribe(seq: str) -> str:
+    res1 = []
+    for char in seq:
+        res1.append(transcribe_dict[char] if char in transcribe_dict else char)
+        res2 = "".join(res1)
+    return res2
```

Comment:
Лучше вывести сборку ответа за пределы цикла, чтобы не собирать строку каждый раз. В остальном все супер!

```suggestion
    return "".join(res1)